lib/locales/hu.ts

```diff
@@ -622,9 +622,9 @@ export const locale: Locale = {
     marcellinus_of_rome_and_peter_the_exorcist_martyrs: 'Saints Marcellinus and Peter, Martyrs',
     marcellus_of_paris_bishop: 'Saint Marcellus of Paris, Bishop',
     margaret_clitherow_anne_line_and_margaret_ward_virgin_martyrs: 'Saints Margaret Clitherow, Anne Line and Margaret Ward, Virgin, Martyrs',
-    margaret_mary_alacoque_virgin: 'Saint Margaret Mary Alacoque, Virgin',
-    margaret_of_antioch_virgin: 'Saint Margaret of Antioch, Virgin and Martyr',
-    margaret_of_hungary_religious: 'Saint Margaret of Hungary, Religious',
+    margaret_mary_alacoque_virgin: 'Saint Margaret Mary Alacoque, Virgin',*/
+    margaret_of_antioch_virgin: 'Antiokheai Szent Margit, szűz és vértanú',
```

Comment:
@MMate2007, where do you found this `Antiokheai` word? It seems to be incorrect.

On 17 Oct, we celebrate Saint Ignatius of Antioch which is translated as `Antiochiai Szent Ignác püspök és vértanú` in both the [Roman Missal](https://drive.google.com/file/d/1vnk7PSgzqn9Kx3yYpGTJK0DgpdinBQtW/view) (ed typ 2) and the [Ordo 2022](https://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=166).

Also, in Hungarian, there is no comma before the titles.

```suggestion
    margaret_of_antioch_virgin: 'Antiochiai Szent Margit szűz és vértanú',
```